draft-tuexen-tsvwg-rfc4960-errata.xml

```diff
@@ -695,6 +695,46 @@ expiration.</t>
 </section>
 </section>
 
+<section title='Order of Adjustments of partial_bytes_acked and cwnd'>
+<!-- The correction is in accordance with RFC 3465, Section 2.1 -->
+<section title='Description of the Problem'>
+<t>Section 7.2.2 of <xref target="RFC4960"/> is unclear about the order 
+of adjustments applied to partial_bytes_acked and cwnd in the Congestion
+Avoidance phase.</t>
+</section>
+<section title='Text Changes to the Document'>
+<figure>
+<artwork>
+---------
+Old text: (Section 7.2.2)
+---------
+
+o  When partial_bytes_acked is equal to or greater than cwnd and
+   before the arrival of the SACK the sender had cwnd or more bytes
+   of data outstanding (i.e., before arrival of the SACK, flightsize
+   was greater than or equal to cwnd), increase cwnd by MTU, and
+   reset partial_bytes_acked to (partial_bytes_acked - cwnd).
+
+---------
+New text: (Section 7.2.2)
+---------
+
+o  When partial_bytes_acked is equal to or greater than cwnd and
+   before the arrival of the SACK the sender had cwnd or more bytes
+   of data outstanding (i.e., before arrival of the SACK, flightsize
+   was greater than or equal to cwnd), partial_bytes_acked is reset
+   to (partial_bytes_acked - cwnd). Next, cwnd is increased by MTU
+   with the limitation that the cwnd shall not be increased by more
+   than one MTU per RTT.
```

Comment:
Yes, it is caused by high pba. 
I think the most obvious use case is when flightsize and cwnd were high and then the loss was detected so that cwnd became low. After that, arrival of SACKs could increase cwnd more often than once per RTT. 
Another use case is when pba was increased on a low traffic rate but cwnd was not.
It is important to state that SCTP MUST NOT increase cwnd by more than 1 MTU per RTT. I believe TCP RFC(s) state the same.
Regarding pba, I think it is still better to decrease it by cwnd every time it becomes equal or greater than cwnd but I need to think more about it (the opposite argument may be that the same is not done by SCTP when SACK acknowledges new data but flightsize is less than cwnd).